pulumi-kubernetes-operator.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: pulumi-kubernetes-operator
+  version: 1.11.5
+  epoch: 0
+  description: A Kubernetes Operator that automates the deployment of Pulumi Stacks
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - go
+      - busybox
+      - git
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/pulumi/pulumi-kubernetes-operator.git
+      tag: v${{package.version}}
+      destination: ${{package.name}}
+      expected-commit: e3f9416a288067828cba13813989aa8a6d5cb9eb
+
+  - working-directory: ${{package.name}}
+    pipeline:
+      - runs: |
+          # Original Go build args found in ./scripts/build.sh
+          CGO_ENABLED=0 go build -o "${{targets.destdir}}/usr/bin/${{package.name}}" \
```

Comment:
you might need to bump up the github.com/prometheus/client_golang to v1.11.1

Reply:
im gonna follow up with this if necessary

Reply:
ok, added